corehq/apps/enterprise/templates/enterprise/partials/project_tile.html

```diff
@@ -8,10 +8,20 @@
     <h4 class="fs-5 pt-2">{{ report.title }}</h4>
   </div>
   <div class="card-body pb-1">
-    <div class="js-total fw-bold fs-1">
-      <i class="fa fa-spin fa-spinner"></i>
+    <div class="row justify-content-center">
+      <div class="col-auto align-items-center position-relative">
+        <div class="js-total fw-bold fs-1">
+          <i class="fa fa-spin fa-spinner"></i>
+        </div>
+        {% if report.total_description %}
+          <div class="position-absolute start-100" style="top: 25%">
+            <span class="hq-help-template" data-content="{{ report.total_description }}" data-placement="right">
+              <i class="fa fa-info-circle"></i>
```

Comment:
What would you think of putting the icon next to the header text instead of the number? It's more secondary/tertiary information than primary information.

Reply:
I feel putting the icon next to the header text implies this description is to explain the header text, not the stat there anymore. But maybe I can move the icon to next to the header text, and then in the tooltip, I included a description of the stat, also a description of the report they will get. I'll bring it up with saas-product. Thank you!